### PR TITLE
Finish database script

### PR DESCRIPTION
**Pull request by techdeus**
@gkim795 Check out my seed script. I am curious about a few aspects of it. Let's talk today.

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+const faker = require('faker');
+const db = require('./index.js');
+const Promise = require('bluebird');
+
+const generator = () => {
+    let result = [];
+
+    for ( var i = 0; i < 100; i++ ) {
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
    for (let i = 0; i < 100; i += 1) {
```

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        let random = Math.floor(Math.random() * Math.floor(100));
+        let details = {
+            plays: faker.random.number(),
+            likes: faker.random.number(),
+            reposts: faker.random.number(),
+            name: faker.name.findName(),
+            imageURL: faker.image.people(),
+            text: faker.lorem.sentence(),
+            createdAt: faker.date.recent(),
+            songtime: faker.random.number({min: 0, max: 300}),
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
            songtime: faker.random.number({ min: 0, max: 300 }),
```

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            comment_Id: faker.random.number({min: 0, max: 100}),
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
            comment_Id: faker.random.number({ min: 0, max: 100 }),
```

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            artist_Id: faker.random.number({min: 0, max: 10}),
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
            artist_Id: faker.random.number({ min: 0, max: 10 }),
```

File: database/seed.js [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            song_Id: faker.random.number({min: 0, max: 10})
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
            song_Id: faker.random.number({ min: 0, max: 10 }) 
```

File: database/seed.js [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            text: faker.lorem.sentence(),
+            createdAt: faker.date.recent(),
+            songtime: faker.random.number({min: 0, max: 300}),
+            comment_Id: faker.random.number({min: 0, max: 100}),
+            artist_Id: faker.random.number({min: 0, max: 10}),
+            song_Id: faker.random.number({min: 0, max: 10}) 
+        }
+        result.push(details);
+    }
+    console.log(result);
```
Comment (lfelberg) [inline database/seed.js]: Remove console.log
```suggestion
```

File: database/seed.js [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+    return result;
+}
+
+var gen = Promise.resolve(generator());
+gen.then( (seeded) => {
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
gen.then((seeded) => {
```

File: database/seed.js [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    let item = seeded;
+
+    for ( let i in seeded ) {
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
    for (let i in seeded) {
```

File: database/seed.js [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+        
+        let current = seeded[i];
+
+        var queryStringSongs = 'INSERT INTO songs (plays, likes, reposts) values (?, ?, ?)';
+
+        let postSongs = [
+            current.plays, 
+            current.likes, 
+            current.reposts
+        ]
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
        ];
```

File: database/seed.js [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+var gen = Promise.resolve(generator());
+gen.then( (seeded) => {
+    let item = seeded;
+
+    for ( let i in seeded ) {
+        
+        let current = seeded[i];
+
+        var queryStringSongs = 'INSERT INTO songs (plays, likes, reposts) values (?, ?, ?)';
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
        const queryStringSongs = 'INSERT INTO songs (plays, likes, reposts) values (?, ?, ?)';
```

File: database/seed.js [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+            if (error) {
+                console.log(error.message);
+            } else {
+                console.log('Data persisted!');
+            }
+        });
+    }
+}).catch(error => {
+    console.log(error);
+});
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
});

```

File: database/seed.js [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+        var queryStringReply = 'INSERT INTO reply (text, createdAt, comment_Id, artist_Id, song_Id) values (?, ?, ?, ?, ?)';
+
+        let postReply = [
+            current.text,
+            current.createdAt,
+            current.songtime,
+            current.comment_Id,
+            current.artist_Id,
+            current.song_Id
+        ]
```
Comment (lfelberg) [inline database/seed.js]:
```suggestion
        ];
```

File: database/seed.js (hunk 1 above)
Comment (techdeus) [reply database/seed.js]: Done!

**Comment by techdeus** [comment]
All changes have been applied! Thank you.

**Pull request merged**